test/end-to-end/employees/registration.page.js

```diff
@@ -114,10 +114,14 @@ class RegistrationPage {
   }
 
   // Set RubricPayroll defined value By Employee
-  setRubricPayroll(rubrics) {
+  async setRubricPayroll(rubrics) {
     const keys = Object.keys(rubrics);
-    const promises = keys.map(key => element(by.id(key)).sendKeys(rubrics[key]));
-    return Promise.all(promises);
+
+    // eslint-disable-next-line
+    for (const key of keys) {
+      // eslint-disable-next-line
+      await element(by.id(key)).sendKeys(rubrics[key]);
```

Comment:
it's really worth removing `Promise.all` for `FU.series` ?

Reply:
There are a few places `Promise.all()` makes sense - when you are reading information, but not doing any actions.  For example, the place it might makes sense is :
```js
const allTextInParagraphs = Promise.all($$('p').map(element => element.getText()));
```
Here, you are only reading the DOM and it doesn't matter what order things happen in.  However, if you are doing a `.click()` or a `.sendKeys()` or any of that, it probably matters what order you do things in.  It also makes more sense since a user can only do one thing at a time.